android/libraries/rib-base/src/main/kotlin/com/uber/rib/core/WorkerBinder.kt

```diff
@@ -59,6 +59,46 @@ public object WorkerBinder {
     this.workerBinderListenerWeakRef = WeakReference<WorkerBinderListener>(workerBinderListener)
   }
 
+  /**
+   * Bind a worker (ie. a manager or any other class that needs an interactor's lifecycle) to an
+   * interactor's lifecycle events
+   *
+   * IMPORTANT: Binding will happen on [RibDispatchers.DEFAULT] unless the Worker explicitly
+   * overrides [coroutineContext] to other than [EmptyCoroutineContext]
+   *
+   * Inject this class into your interactor and call this method on any
+   *
+   * @param interactor The interactor that provides the lifecycle.
+   * @param worker The class that wants to be informed when to start and stop doing work.
+   * @return [WorkerUnbinder] to unbind [Worker]'s lifecycle.
+   */
+  @JvmStatic
+  public fun bindToInteractor(
+    interactor: Interactor<*, *>,
+    worker: Worker,
```

Comment:
Do you feel like we should provide an optional `dispatcher` parameter like we had before, but defaulting to `RibDispatchers.Default` instead?

```
  @JvmStatic @JvmOverloads
  public fun bindToInteractor(
    interactor: Interactor<*, *>,
    worker: Worker,
    dispatcher: CoroutineDispatcher = RibDispatchers.Default,
  ): WorkerUnbinder = bind(interactor, worker, dispatcher)
```

Reply:
I guess we could keep it but at the same time I'm not too sold on this given the worker can specify its own dispatcher to override the one passed at WorkerBinder. 

If there is a need to specify another dispatcher probably is best to define within Worker level for clarification and to prevent unwanted changes outside the worker definition

Reply:
I'm fine with that